lib/simple_server_extensions.rb

```diff
@@ -0,0 +1,23 @@
+module SimpleServerExtensions
+  # We first check for the REVISION file that Capistrano deploys for us, otherwise we fall
+  # back to using git to grab it from the live repo (for dev/test environments)
+  def self.determine_git_ref
+    if Rails.root.join("REVISION").exist?
+      Rails.root.join("REVISION").read
+    else
+      `git rev-parse HEAD`.chomp
+    end
+  end
+
+  GIT_REF = determine_git_ref
+
+  def env
+    ActiveSupport::StringInquirer.new(SIMPLE_SERVER_ENV)
```

Comment:
@simpledotorg/server-developers This will give us a nicer way to get to the often used SIMPLE_SERVER_ENV:

```
SimpleServer.env.production?
SimpleServer.env.test?
etc
```